vcd/resource_vcd_lb_app_rule_test.go

```diff
@@ -0,0 +1,194 @@
+// +build gateway lb lbAppRule ALL functional
+
+package vcd
+
+import (
+	"fmt"
+	"html/template"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccVcdLBAppRule(t *testing.T) {
+	// The Script parameter must be sent as multiline string separated by \n characters.
```

Comment:
can we say "newlines" instead of "\n characters"?

Reply:
Let's leave `\n` as it's instantly clear, while newline has some blurry aspects to it (you need to digest that this is \n first, and even then you may think (if you come from Windows background) does it mean \n or \r\n...

Reply:
I can mention both

Reply:
Done.